pkg/controller/labelidentity/label_group_index.go

```diff
@@ -299,45 +313,26 @@ func (i *LabelIdentityIndex) deleteSelector(selectorKey string, policyKey string
 	}
 }
 
-// SetPolicySelectors registers ClusterSet-scope policy selectors with the labelIdentityIndex,
-// and then retrieves all the LabelIdentity IDs that currently match these selectors.
-func (i *LabelIdentityIndex) SetPolicySelectors(selectors []*types.GroupSelector, policyKey string) []uint32 {
-	var labelIdentityIDs []uint32
+// SetPolicySelectors registers a policy's latest selectors with the index.
+// For the selectors that the policy no longer possess, it removes the policy key
+// from those selectorItems.
+func (i *LabelIdentityIndex) SetPolicySelectors(selectors []*types.GroupSelector, policyKey string) {
 	newSelectors := map[string]*types.GroupSelector{}
 	for _, s := range selectors {
-		klog.V(4).InfoS("Getting matched LabelIdentity for policy selector", "selector", s.NormalizedName, "policy", policyKey)
 		newSelectors[s.NormalizedName] = s
 	}
 	originalSelectors := i.getPolicySelectors(policyKey)
-	for selKey, sel := range newSelectors {
+	for selKey := range newSelectors {
```

Comment:
I just see this func delete selectors, but do not see it adds new selectors. Where they are handled? It semes not match the func name and description.

Reply:
You are right... the function name is very outdated. Adding new selectors is handled by `AddSelectors` and this function only cleans up stale selectors. Updated the naming and comments accordingly.

Reply:
I did not see where `AddSelectors` is defined/called. Could you point to me the code?

Reply:
@Dyanngg : reminder.

Reply:
In the function `toAntreaPeerForCRD`:
```
...
if n.stretchNPEnabled && peer.Scope == v1alpha1.ScopeClusterSet {
        newClusterSetScopeSelector := antreatypes.NewGroupSelector(np.GetNamespace(), peer.PodSelector, peer.NamespaceSelector, nil, nil)
	*clusterSetScopeSelectors = append(*clusterSetScopeSelectors, newClusterSetScopeSelector)
	labelIdentities = append(labelIdentities, n.labelIdentityInterface.AddSelector(newClusterSetScopeSelector, internalNetworkPolicyKeyFunc(np))...)
}
...
```
This is called for each regular ClusterSet scoped selector, or computed per-Namespace selector that have scope: ClusterSet